packages/sx05re/emuelec/bin/emuelec_autostart.sh

```diff
@@ -16,20 +16,13 @@ if [ ! -L "$CONFIG_DIR" ]; then
 ln -sf $CONFIG_DIR2 $CONFIG_DIR
 fi
 
-if [ "${EE_DEVICE}" == "Amlogic" ]; then
-  rm /storage/.config/asound.conf > /dev/null 2>&1
-  cp /storage/.config/asound.conf-amlogic /storage/.config/asound.conf
-
-    if [ "$(get_es_setting bool StopMusicOnScreenSaver)" != "false" ]; then 
-        sed -i "/<bool name=\"StopMusicOnScreenSaver.*/d" "${ES_CONF}"
-        sed -i "s|</config>|	<bool name=\"StopMusicOnScreenSaver\" value=\"false\" />\n</config>|g" "${ES_CONF}"
-    fi
-
-elif [ "${EE_DEVICE}" == "Amlogic-ng" ]; then
-  rm /storage/.config/asound.conf > /dev/null 2>&1
-  cp /storage/.config/asound.conf-amlogic-ng /storage/.config/asound.conf
+if [ "$(get_es_setting bool StopMusicOnScreenSaver)" != "false" ]; then 
```

Comment:
What does this have to do with this PR?

Reply:
wait a sec, there is a better way to this, give me about half an hour.

Reply:
Okay, you can check it now. The screensaver stuff is legacy code for amlogic devices, do we need it? Basically the default asound device file only gets copied to asound.conf if it doesn't exist. and the utils function just now works on asound.conf. 

Good for you?

Reply:
one more thing, should we also in setauddev() set the retroarch config audio_device? Allot of people have no idea how to set that via menu.